packages/commonwealth/server/routes/threads/update_thread_handler.ts

```diff
@@ -26,7 +26,7 @@ type UpdateThreadRequestBody = {
   canvasSession?: any;
   canvasAction?: any;
   canvasHash?: any;
-  discordMeta?: any;
+  discord_meta?: any; // Only comes from the discord bot
```

Comment:
Why do we need to rename this? Breaks the convention. @alexyoung23j

Reply:
Well, this was kinda just a judgement call. The consumer and listener were implemented with snake casing in everything, and the only place it conflicts was on this param. I figured it was better to just change this one line, but we can go back and change all of the consumer/listener code and the other routes they touch if you prefer.

Reply:
@alexyoung23j please open a ticket and we can discuss on the ticket how we want to do naming on the backend.